webrtc-stats.html

```diff
@@ -1554,9 +1554,17 @@ <h2>
               </dt>
               <dd>
                 <p>
+                  The purpose of the jitter buffer is to recombine RTP packets into frames (in the case of video)
+                  and have smooth playout. The model described here assumes that the samples or frames are 
+                  still compressed and have not yet been decoded.
                   It is the sum of the time, in seconds, each <a>audio sample</a> or video frame takes from
-                  the time it is received and to the time it exits the jitter buffer. This increases
-                  upon samples or frames exiting, having completed their time in the buffer (and
+                  the time it the first packet received by the jitter buffer (ingest timestamp) and to the 
+                  time it exits the jitter buffer (emit timestamp). 
```

Comment:
Do we need to clarify "to the time [the last packet of that frame] exits the jitter buffer"?